src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -458,30 +459,56 @@ csv::Windows1252Parser Dataloader::parse_csv(fs::path const& path) {
 }
 
 bool Dataloader::_load_pop_types(PopManager& pop_manager, UnitManager const& unit_manager, GoodManager const& good_manager, fs::path const& pop_type_directory) const {
-	const bool ret = apply_to_files_in_dir(pop_type_directory, ".txt",
-		[&pop_manager, &unit_manager, &good_manager](fs::path const& file) -> bool {
-			return pop_manager.load_pop_type_file(file.stem().string(), unit_manager, good_manager, parse_defines(file).get_file_node());
-		}
-	);
+	const bool ret = apply_to_files_in_dir(pop_type_directory, ".txt", [&pop_manager, &unit_manager, &good_manager](fs::path const& file) -> bool {
+		return pop_manager.load_pop_type_file(file.stem().string(), unit_manager, good_manager, parse_defines(file).get_file_node());
+	});
 	pop_manager.lock_pop_types();
 	return ret;
 }
 
 bool Dataloader::_load_units(UnitManager& unit_manager, GoodManager const& good_manager, fs::path const& units_directory) const {
-	const bool ret = apply_to_files_in_dir(units_directory, ".txt",
-		[&unit_manager, &good_manager](fs::path const& file) -> bool {
-			return unit_manager.load_unit_file(good_manager, parse_defines(file).get_file_node());
-		}
-	);
+	const bool ret = apply_to_files_in_dir(units_directory, ".txt", [&unit_manager, &good_manager](fs::path const& file) -> bool {
+		return unit_manager.load_unit_file(good_manager, parse_defines(file).get_file_node());
+	});
 	unit_manager.lock_units();
 	return ret;
 }
 
-bool Dataloader::_load_countries(GameManager& game_manager, fs::path const& countries_file, ast::NodeCPtr root) const {
+bool Dataloader::_load_oobs(GameManager& game_manager) const {
+	static const fs::path oob_directory = "history/units";
+
+	/* used for countries with no defined initial OOB */
+	game_manager.get_military_manager().get_order_of_battle_manager().add_oob("NULL", std::move(std::vector<Army>()), std::move(std::vector<Navy>()), std::move(std::vector<Leader>()));
```

Comment:
std::move here is completely superfluous here, construction for rvalues automatically makes them an rvalue, move should only really be useful for moving an lvalue, usually an already constructed temporary, into an rvalue.